tests/testDetector.py

```diff
@@ -217,6 +217,36 @@ def testGetCenter(self):
                 if cameraSys == cameraGeom.PIXELS:
                     self.assertAlmostEquals(ctrPixPoint[i], ctrPoint[i])
 
+    def testCopyDetector(self):
+        """Test copyDetector() method
+        """
+        #
+        # Make a copy without any modifications
+        #
+        detector  = DetectorWrapper().detector
+        ndetector = cameraGeom.copyDetector(detector)
+
+        assert detector.getName() == ndetector.getName()
```

Comment:
Please use `self.assertEquals` and such, so that test failures show up in the usual fashion. Likewise for testPropertyListPersistence.py below.

Also please consider renaming tests from `testFoo.py` to `test_foo.py` when you work on them. Thats the new standard (to aid automatic discovery by pytest).